docs/api-routes/introduction.md

```diff
@@ -18,7 +18,7 @@ API routes provide a solution to build your **API** with Next.js.
 
 Any file inside the folder `pages/api` is mapped to `/api/*` and will be treated as an API endpoint instead of a `page`. They are server-side only bundles and won't increase your client-side bundle size.
 
-For example, the following API route `pages/api/user.js` returns a `json` response with a status code of `200`:
```

Comment:
This was the file name. It is necessary to explain the name of the file for the code snippet below.

Reply:
Sure. I do understand that it is necessary to explain the name of the file. But for the API route, I thought it might be confusing. Thank you.